db/config.c

```diff
@@ -234,24 +234,27 @@ static int lrl_if(char **tok_inout, char *line, int line_len, int *st,
     return 1; /* there was no "if" statement or it was true */
 }
 
-int process_deferred_options(struct dbenv *dbenv,
```

Comment:
I'd keep this and check the return code of process_command() in deferred_do_commands() and pre_read_option() in pre_read_deferred_callback(), instead.